underscore.js

```diff
@@ -1279,9 +1288,7 @@
 
   // Is a given value an array?
   // Delegates to ECMA5's native Array.isArray
-  _.isArray = nativeIsArray || function(obj) {
-    return toString.call(obj) === '[object Array]';
-  };
+  _.isArray = isArray;
```

Comment:
@akre54 I'm not sure I understand. If we move `_.property` below where the current `_.is[...]` methods live, it' won't be instantiated when we go to create `getLength`.